library/lua/test/expect.lua

```diff
@@ -101,25 +101,130 @@ function expect.table_eq(a, b, comment)
             ('key %s: "%s" ~= "%s"'):format(keystr, diff[1], diff[2])
 end
 
-function expect.error(func, ...)
-    local ok, ret = pcall(func, ...)
+function expect.error(func, comment)
```

Comment:
+1 on this change. I [previously mentioned](https://github.com/DFHack/scripts/pull/263#discussion_r603772718) `expect.eq` taking `...` params, but it actually takes a single `comment` param too (as I'm sure you noticed). Much simpler to deal with here.